src/styles/block/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+  justify-content: center;
+
+  &__item {
+    width: 16px;
+    height: 16px;
+    background-image: url(/src/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+  }
+}
+
+.stars--4 .stars__item:nth-child(-n + 4) {
```

Comment:
Ensure that the BEM naming convention aligns with the HTML structure. The modifier `stars--4` is used correctly, but verify that it matches the intended use in the HTML file.